eslint.config.ts

```diff
@@ -0,0 +1,71 @@
+import { FlatCompat } from "@eslint/eslintrc";
+import eslint from "@eslint/js";
+import comments from "@eslint-community/eslint-plugin-eslint-comments";
```

Comment:
I get an info warning on this line:
```
Could not find a declaration file for module '@eslint-community/eslint-plugin-eslint-comments'.
'roblox-ts/node_modules/@eslint-community/eslint-plugin-eslint-comments/index.js' implicitly has an 'any' type.
```
is this intentional?

Reply:
I couldn't find a `@types` package for this and didn't think it was worth writing one manually.